cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/xcm_config.rs

```diff
@@ -489,6 +460,25 @@ impl<
 	}
 }
 
+pub struct XcmFeeManagerFromComponentsBridgeHub<WaivedLocations, HandleFee>(
+	PhantomData<(WaivedLocations, HandleFee)>,
+);
+impl<WaivedLocations: Contains<MultiLocation>, FeeHandler: HandleFee> FeeManager
+	for XcmFeeManagerFromComponentsBridgeHub<WaivedLocations, FeeHandler>
+{
+	fn is_waived(origin: Option<&MultiLocation>, fee_reason: FeeReason) -> bool {
+		let Some(loc) = origin else { return false };
+		if let Export { network: Ethereum { chain_id: 15 }, destination: Here } = fee_reason {
+			return false
+		}
+		WaivedLocations::contains(loc)
+	}
+
+	fn handle_fee(fee: MultiAssets, context: Option<&XcmContext>, reason: FeeReason) {
+		FeeHandler::handle_fee(fee, context, reason);
+	}
+}
+
```

Comment:
In this latest version, fees from asset hub to bridgehub are waived. There was a bug in the previous version, which is why it is only failing now. Introduced `XcmFeeManagerFromComponentsBridgeHub` to check if the fee reason is `Export` and from the Ethereum network. If so, a fee needs to be paid.

Let me know if you think this is OK and if I should check if the origin is from Asset hub.